.travis.yml

```diff
@@ -1,3 +1,3 @@
 language: objective-c
-osx_image: xcode6.4
+osx_image: xcode7.1
```

Comment:
Oh blimey! I didn't even notice that had come out.
No worries. I'll bump it to 7.1.1 then!